itests/eth_transactions_test.go

```diff
@@ -628,12 +628,6 @@ func TestTraceTransaction(t *testing.T) {
 	require.Contains(t, err.Error(), "transaction not found")
 	require.Nil(t, traces)
 
-	// EthTraceTransaction errors when a trace for pending transactions is requested
```

Comment:
why removed?

Reply:
Because this is racy. It is possible that the transaction has already been mined by the time we request this.

Reply:
oh, right, true!